### DISCUSSION TITLE
[Bug] The Designer width switches to 992px when copy pasting an element

### DISCUSSION THREAD

**Discussion by Rehankhurshid** [webstudio-is/webstudio-community#30] [Q&A]
Whenever I copy-paste an element, the designer sw to the 992px breakpoint on all the breakpoint whether it will be Desktop (let's say 1440px), Tablet & Mobile.
<img width="1148" alt="image" src="https://github.com/webstudio-is/webstudio/assets/24763068/89e84f7c-ace9-439b-85d7-ed33e9a52b91">

**Comment by kof** [comment]
Amazing, thanks for the report

**Comment by TrySound** [comment]
Here's the fix https://github.com/webstudio-is/webstudio-builder/pull/1631
